src/audio-worklet/free-queue/src/freequeue.mjs

```diff
@@ -61,11 +44,15 @@ export class FreeQueue {
     }
   }
 
-  static fromPointers(fqPointers: FreeQueuePointers) {
+  /**
+   * Helper function for creating FreeQueue from pointers.
+   * @param fqPointers
+   * @returns FreeQueue
+   */
+  static fromPointers(fqPointers) {
```

Comment:
"pointers" - as in plural?

So does it take multiple pointers?

Reply:
Yup, for now!
I developed this approach for initial testing and proof of concept.

example - https://github.com/DivyamAhuja/free_queue_c/blob/b985cab58c6b0cd8ca46a20952be1e003ab84b02/js/worker.js#L7-L25

I wonder if you have some nice idea. (I was thinking we can pointers just by doing pointer arithmetic and offsets, but was afraid of wasm32 vs wasm64 thing if it exist. and I think this approach makes updating code and API at any point much easier)